### PR TITLE
Fix: #6674 HeadRenderer: When Session tracking is 'URL', HeadRenderer don't add the jsesionid param to the resources #6674

### LINKED ISSUE #6674: HeadRenderer: When Session tracking is 'URL', HeadRenderer don't add the jsesionid param to the resources
**Reproducer**
When session tracking mode is 'URL', the jsesionid must be appended to the resources url.
HeadRenderer don't append the jsessionid to the css and validation resources, then new session is created on every request.

**Environment:**
 - PF Version: all
 - JSF + version: all

**Expected behavior**
HeadRenderer must encode resources to append jsessionid

### PR DESCRIPTION
**Pull request by dgoicocheta**
Fixes: #6674 
The solution is simple 
Tested on mojarra 2.2, 2.3 and  myfaces 2.2, 2.3

### CODE REVIEW

**Review by melloware — APPROVED** [review]

**Comment by melloware** [comment]
Looks good to me I will wait until @tandraschko reviews and then merge it.

**Comment by tandraschko** [comment]
Thanks!

**Pull request merged**
